src/components/Button/index.tsx

```diff
@@ -239,9 +239,23 @@ export const Button = ({
     >
       <Component href={href || ''} target={target}>
         {webProperty === 'ousd' || webProperty === 'oeth' ? (
-          <div className={twMerge(bg, isButton && 'rounded-full px-8 py-1')}>
-            {label}
-            {children}
+          <div
+            className={`relative bg-gradient2 rounded-[100px] w-full md:w-fit h-fit ${
+              isButton && 'hover:opacity-90'
```

Comment:
nit: when using `&&` as an conditional operator when constructing strings, it'll insert the stringified value of the constraint if it's falsy

Reply:
Yeah noticed that... it inserts `undefined`. I'll change it to ternary operator.

Reply:
Agreed this is why I like using something like `classnames` to do conditional class merge, there may be a built-in for tailwind merge that does this?

The classnames would look like,
```
classNames("relative bg-gradient2 rounded-[100px] w-full md:w-fit h-fit", {'hover:opacity-90': isButton })'
```